src/scss/_vw_l.scss

```
@@ -0,0 +1,15 @@
+.Florence {
+  font-family: "Noto Sans JP", sans-serif;
+    &__footer {
+      font-family: "Roboto", sans-serif;
+      @include center;
+      height: 60px;
+      font-size: 12px;
+      box-shadow: -2px -7px 20px -5px rgb(rgb(189, 189, 189), 0.2);
```

```suggestion
      box-shadow: -2px -7px 20px -5px rgba(189, 189, 189, 0.2);
```

rgbaというパラメータがあるのでこちらを使用しましょう

---

```suggestion
      box-shadow: -2px -7px 20px -5px rgba(#bdbdbd, 0.2);
```

SCSSで書くなら上記のように16進数カラーコードで書くこともできます。
カラーはカラー定義するファイルに変数として保存しておくと再利用性が増してメンテナンスしやすいコードになりと思います。